File-Formats/Presentation/Presentation-to-image.md

```diff
@@ -574,29 +574,33 @@ using (FileStream fileStreamInput = new FileStream("Template.pptx", FileMode.Ope
 
 {% endtabs %}
 
-You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PowerPoint-Examples/tree/master/PPTX-to-Image-conversion/Apply-fallback-fonts).
+### Fallback fonts based on script type
 
-### Customize Default Fallback Fonts
-
-The following code example demonstrates how to customize default fallback font while converting a PowerPoint presentation to Image.
+The following code example demonstrates how a user can add fallback fonts based on the script types, which Presentation considers internally when converting a PowerPoint presentation to Image.
 
 {% tabs %}
 
-{% highlight c# tabtitle="C# [Cross-platform]" %}
-
-//Load the PowerPoint presentation into stream.
-using (FileStream fileStreamInput = new FileStream(@"Template.pptx", FileMode.Open, FileAccess.Read))
+{% highlight C# tabtitle="C# [Cross-platform]" %}
+//Load the PowerPoint presentation into stream
+using (FileStream fileStreamInput = new FileStream("Template.pptx", FileMode.Open, FileAccess.Read))
 {
     //Open the existing PowerPoint presentation with loaded stream
     using (IPresentation pptxDoc = Presentation.Open(fileStreamInput))
     {
-        //Initialize the PresentationRenderer to perform image conversion
-        pptxDoc.PresentationRenderer = new PresentationRenderer();
-        //Use a sets of default FallbackFont collection to IPresentation
-        pptxDoc.FontSettings.InitializeFallbackFonts();
-        // Customize a default fallback font name
-        // Modify the Hebrew script default font name as "David"
-        pptxDoc.FontSettings.FallbackFonts[5].FontNames = "David";
+        //Adds fallback font for "Arabic" script type
```

Comment:
//Adds fallback font for "Arabic" script type.

Reply:
Added pull stop